src/dbs/init.mysql.js

```diff
@@ -36,21 +36,31 @@ class Database
         return Database.instance
     }
 
+	releaseAllConnection()
+	{
+		if(this.typeConnection == TYPE_CONNECTION_DB.pool)
+		{
+			this.pool.end()
+		}
```

Comment:
should check the pool is null or not before end()

```suggestion
		if(this.typeConnection == TYPE_CONNECTION_DB.pool && this.pool)
		{
			this.pool.end()
		}
```

Reply:
Thanks I will update in the next commit